<launch>
    <arg name="topic_name" default="/qsrlib_rviz" />

    <arg name="machine" default="localhost" />
    <arg name="user" default="" />

    <machine name="$(arg machine)" address="$(arg machine)" env-loader="$(optenv ROS_ENV_LOADER )" user="$(arg user)" default="true"/>
    
    <node pkg="qsr_lib" type="qsrlib_ros_server.py" name="qsr_lib" output="screen"/>

    <node pkg="qsr_lib" type="rviz_qsrlib_server.py" name="qsrlib_rviz_server" output="screen">
	<param name="topic_name" value="$(arg topic_name)" type="string"/>
   </node>
</launch> 
